content/css/mobile-styles.scss

```diff
@@ -29,7 +29,7 @@
   }
 
   .banner {
-    background-image: url("/imgs/noetic-background.png");
+    background-image: url("/imgs/JazzyNoBG.png");
```

Comment:
<img width="434" height="942" alt="image" src="https://github.com/user-attachments/assets/d1aa87dc-87dd-450e-a4ed-1e9fb931c784" />
<img width="434" height="942" alt="Screenshot From 2025-07-11 13-36-32" src="https://github.com/user-attachments/assets/057d2db7-6843-4200-81f8-f967e7eadfc7" />

I would propose we just drop the image on mobile. It makes it really hard to render the text.

Reply:
Sounds good.  I've now removed it.